www/views/includes/txp.html

```diff
@@ -32,8 +32,7 @@
       </span>
     </span>
     <div>
-      <time class="wallet-details__tx-time" ng-if="tx.createdOn && createdWithinPastDay(tx.createdOn)">{{tx.createdOn * 1000 | amTimeAgo}}</time>
-      <time class="wallet-details__tx-time" ng-if="tx.createdOn && !createdWithinPastDay(tx.createdOn)">{{tx.createdOn * 1000 | date:'MMMM d, y'}}</time>
```

Comment:
`createdWithinPastDay` function missing on _tab-home.js_ and _proposals.js_. Already added 👍